### PR TITLE
fix: no query_range api call on layout change

### PR DESCRIPTION
**Pull request by Rajat-Dabade**
Close https://github.com/SigNoz/signoz/issues/2797

### CODE REVIEW

File: frontend/src/container/GridGraphLayout/index.tsx [hunk 1]
```diff
@@ -217,28 +217,28 @@ function GridGraph(props: Props): JSX.Element {
...
-								yAxisUnit={currentWidget?.yAxisUnit}
-								layout={layout}
-								setLayout={setLayout}
-								onDragSelect={onDragSelect}
-							/>
-						),
-					};
-				}),
-			);
+			for (let i = 0; i < layout.length; i += 1) {
```
Comment (palashgdev) [inline frontend/src/container/GridGraphLayout/index.tsx:220]:
```suggestion
const setLayoutFunction = useCallback(
	(layout: Layout[]) => {
		layout.forEach((item, i) => {
			const currentWidget =
				widgets?.find((widget) => widget.id === item.i) || ({} as Widgets);
			layouts[i] = {
				...item,
				Component: (): JSX.Element => (
					<Graph
						name={currentWidget.id}
						widget={currentWidget}
						yAxisUnit={currentWidget?.yAxisUnit}
						layout={layout}
						setLayout={setLayout}
						onDragSelect={onDragSelect}
					/>
				),
			};
		});

		setLayout((layouts) => layouts);
	},
	[widgets, onDragSelect, layouts],
);
```

**Review by palashgdev — CHANGES_REQUESTED** [review]

**Comment by makeavish** [comment]
@Rajat-Dabade Is it required to call query range API, when we save the layout?
IMO it's not required as layout change is not related to query range API.
cc: @palashgdev

**Comment by Rajat-Dabade** [comment]
@makeavish, It should not be required, should we do it as a part of this PR, or should create a new issue for that?
Thoughts @palashgdev

**Comment by makeavish** [comment]
> @makeavish, It should not be required, should we do it as a part of this PR, or should create a new issue for that? Thoughts @palashgdev

This is happening after these PR changes and is related to current changes only. On staging we don't make query range API on saving layout. So I think it should be part of this PR.

**Comment by Rajat-Dabade** [comment]
@makeavish, Okay I got it. Looking into it.

**Comment by makeavish** [comment]
I think this PR is obsolete and can be closed, reopen if needed.

**Pull request closed without merging**